CHANGELOG.md

```diff
@@ -6,13 +6,24 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
 
 ## [??? (unreleased)]
 
+
+## [2020.6] - 2020-04-30
+
 ### Added
 
 - libaktualizr API and aktualizr-primary command line parameter to provide custom hardware information in JSON format: [PR](https://github.com/advancedtelematic/aktualizr/pull/1644)
 
 ### Changed
 
 - Improved garage-deploy object fetching performance by reusing the curl handle: [PR](https://github.com/advancedtelematic/aktualizr/pull/1643)
+- Added an SQL busy handler with 2 seconds timeout: [PR](https://github.com/advancedtelematic/aktualizr/pull/1648)
```

Comment:
Maybe these two are not so interesting for users? That's probably ok anyway, the change list is quite small this time.